astropy/cosmology/core.py

```diff
@@ -5,6 +5,7 @@
 from math import acos, sin, cos, sqrt, pi, exp, log, floor
 from abc import ABCMeta, abstractmethod
 from inspect import signature
+from types import MappingProxyType
```

Comment:
Not used right now, I think

Reply:
Good catch. Thanks.

Reply:
R.e. merging ones own PRs... As the only Cosmology maintainer, what's the alternative? I 100% get the need to request review, but isn't the accountability associated with merging supposed to lie with the maintainer?

Reply:
Hmm, not sure - but I do not worry too much about merging in other subpackages... Which reminds me that I may ask you to have another look at #11470 and #11775 at some point...